index.html

```diff
@@ -179,6 +179,24 @@ <h2>About me</h2>
                 </li>
             </ul>
         </section>
+        <footer class="contact-form">
+            <h2>I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xknylodz" method="POST">
+                <input type="text" placeholder="Full name" name="user-name" required maxlength="30">
+                <input type="email" placeholder="Email address" name="user-email" required>
+                <textarea name="user-message" cols="30" rows="5" required maxlength="500">Enter text here</textarea>
```

Comment:
- [ ] Kindly make sure the "Enter text here" is a `placeholder` and not a `text`. Because of this, the user can submit the form without writing anything in the `textarea` 🙏 .